frontend/src/views/tests/taskSelection.test.js

```diff
@@ -86,15 +90,22 @@ describe('Task Selection Page', () => {
   });
 
   it('should change the button text to map selected task when user selects a task', async () => {
-    setup();
+    act(() => {
+      store.dispatch({
+        type: 'SET_USER_DETAILS',
+        userDetails: { id: 69, username: 'user_3', isExpert: true },
+      });
+      store.dispatch({ type: 'SET_TOKEN', token: 'validToken' });
+    });
```

Comment:
was this necessary? since the state has already been assigned to the same value in prior tests

Reply:
I thought I dropped this when I cherry-picked the commit. I'll double-check that. Maybe I missed this one.